pkg/section/local_module.go

```diff
@@ -59,12 +61,21 @@ func findLocalModules() ([]string, error) {
 
 	uniqueModules := make(map[string]struct{})
 
-	for _, pkg := range packages {
-		if len(pkg.Errors) != 0 {
+	for _, pkg := range pkgs {
+		if len(pkg.Errors) != 0 && !errorsContainListError(pkg.Errors) {
 			return nil, fmt.Errorf("error reading local packages: %v", pkg.Errors)
 		}
 		if pkg.Module != nil {
 			uniqueModules[pkg.Module.Path] = struct{}{}
+		} else {
```

Comment:
:thinking: is life easier if we just check for `go.mod` before even listing local packages? I imagine the most common use case is running from the top-level where `go.mod` exists + reading `go.mod` should be cheaper than loading packages (and if we load a module there we could probably just bail out and not even bother listing packages)

Reply:
We can raise this issue in the GO upstream itself.

Reply:
Ok, got it. The "packages" package is intended to deal with packages, not with modules itself. So, there is no issue. Proposed a new commit with go.mod processing.